components/molecules/GameCard/GameCard.tsx

```diff
@@ -0,0 +1,221 @@
+        },
+        miniImageWithSubtitle: {
+          width: 32,
+          height: 32,
+          marginRight: 10,
+          borderRadius: theme.roundness.xs,
+          borderWidth: 0.5,
+          borderColor: theme.colors.neutral + '99',
+        },
+        name: {
+          width: 120,
+          height: 32,
+          color: theme.colors.neutral,
+          marginTop: theme.spacing.md,
+        },
+        notificationLabelContainer: {
+          position: 'absolute',
+          top: 0,
+          right: 0,
+        },
+        miniContainer: {
+          flexDirection: 'row',
+          alignItems: 'center',
+        },
+      }),
+    [theme],
+  )
+
+  let content = <Fragment />
+
+  if (variation === 'full') {
+    content = (
+      <View style={styles.container}>
+        {/* Card Main */}
+        <View style={styles.imageContainer}>
+          <Image
+            source={{ uri: imageUrl }}
+            style={styles.image}
+            resizeMode="cover"
+          />
+          <Title numberOfLines={2} variation="subtitle2" style={styles.name}>
+            {name}
+          </Title>
+        </View>
+
+        {/* Card Footer */}
+        <View style={styles.footerContainer}>
+          <LikeStat likes={likesByFriends!} suffix="friends" />
+          <LikeStat likes={likesOnPlatform!} />
+        </View>
+
+        {notificationsCount > 0 && (
+          <View style={styles.notificationLabelContainer}>
+            <Label count={notificationsCount} />
+          </View>
+        )}
+      </View>
+    )
+  } else if (variation === 'mini') {
+    content = (
+      <View>
+        <Image source={{ uri: imageUrl }} style={styles.image} />
+        <Title numberOfLines={2} variation="subtitle2" style={styles.name}>
+          {name}
+        </Title>
+      </View>
+    )
+  } else if (variation === 'icon') {
+    if (callout) {
+      content = (
+        <View style={styles.miniContainer}>
+          <Image
+            source={{ uri: imageUrl }}
+            style={styles.miniImageWithSubtitle}
+          />
+          <View>
+            <Title variation="subtitle1">{name}</Title>
+            <Text>{callout}</Text>
+          </View>
+        </View>
+      )
+    } else {
+      content = (
+        <View style={styles.miniContainer}>
+          <Image source={{ uri: imageUrl }} style={styles.miniImage} />
+          <Title variation="subtitle1">{name}</Title>
+        </View>
+      )
+    }
+  } else {
+    console.warn(
+      'Invalid Card variation, Should be one of `full` | `icon` | `mini`',
+    )
+    return <View />
+  }
+
+  const handlePress = () => {
+    if (onPress && typeof onPress === 'function') {
+      onPress({
+        name,
+        imageUrl,
+        likesByFriends,
+        likesOnPlatform,
+      })
+    }
+  }
+
+  return (
+    <Pressable
+      onPress={handlePress}
+      style={({ pressed }) => [styles.wrapper, { opacity: pressed ? 0.8 : 1 }]}
+    >
+      {content}
+    </Pressable>
```

Comment:
@nbonatsakis is it okay to use Pressable? We've always been using TouchableRipple

Reply:
`TouchableRipple` is a react native paper component that implements the material-style ripple. Just did some reading, looks like `Pressable` is a more modern component in core react native that supersedes `TouchableHighlight`, `TouchableOpacity`, etc. 

I think we should migrate to using `Pressable` since it's what RN recommends now.